lib/gelf/logger.rb

```diff
@@ -5,7 +5,7 @@ module LoggerCompatibility
     attr_accessor :formatter
 
     # Use it like Logger#add... or better not to use at all.
-    def add(level, message = nil, progname = nil)
+    def add(level, message = nil, progname = nil, &block)
```

Comment:
It makes it clear from the `add` method signature that it takes a block and therefore will presumably do something with it. It's more a documentation convenience for readers than a compatibility issue (I'm not aware of any compatibility issue btw).

Let me know if this doesn't fit the project code style so I can remove it :)